R/estimate_dispersion.R

```diff
@@ -168,20 +168,21 @@ estimate_dispersion <- function(
       start_row <- max_t - d - k + 2
       end_row <- max_t - d + 1
       obs <- trunc_matr_observed[start_row:end_row, ]
+      tri <- triangle_observed[start_row:end_row, ]
       nowcast <- nowcast_i[start_row:end_row, ]
-      indices_nowcast <- is.na(triangle_observed)[start_row:end_row, ]
-      indices_observed <- !is.na(trunc_matr_observed)[start_row:end_row, ] # nolint
+      indices_nowcast <- is.na(tri)
```

Comment:
do you need this assigment complexity given obs and ti are used once? To me this makes the code harder to read/reason on

Reply:
inparticular because the order you define them is no the order you then use them in

Reply:
```suggestion
    indices_nowcast <- is.na(triangle_observed[start_row:end_row, ])
    indices_observed <- !is.na(trunc_matr_observed[start_row:end_row, ])
```
I could just use this?